### ISSUE TITLE
Indexer and scanner processes do not appear to be running on Windows

### ISSUE THREAD

**Issue by dustinbyrne** [applandinc/appmap-intellij-plugin#112]
AppMaps are not indexed or scanned on Windows. None of the expected subprocesses are present in the task manager (assuming they're children of the IDE). No errors are reported.

![Image](https://user-images.githubusercontent.com/8737782/189414605-3e7ba1a8-44cf-4d2d-91b4-591fcd8b8fda.png)

It looks like the binaries are not downloaded. I'd expect binaries to exist in these subdirectories, but they're empty:
```
PS C:\Users\Dustin\AppData\Local\JetBrains\appland-plugin> Get-ChildItem -Recurse


    Directory: C:\Users\Dustin\AppData\Local\JetBrains\appland-plugin


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----          9/9/2022   2:10 PM                appmap
d-----          9/9/2022   2:12 PM                scanner
```

**Comment by jansorg** [comment]
Thanks!
Fixed by https://github.com/applandinc/appmap-intellij-plugin/pull/81/commits/88382162181cf1d0e27bae79bd89a5381cf31aa3
This wasn't found by CI, because it was recently decided on Slack not to run the plugin tests on Windows CI

**Comment by jansorg** [comment]
Fixed with 0.11.0-2, please reopen if it's still a bug on your machine.
https://github.com/applandinc/appmap-intellij-plugin/releases/tag/install-guide-preview
